src/main/java/org/threadly/concurrent/AbstractPriorityScheduler.java

```diff
@@ -17,17 +17,21 @@
 import org.threadly.util.ExceptionUtils;
 
 /**
- * <p>Abstract implementation for implementations of {@link PrioritySchedulerInterface}.  This likely 
- * is not useful to many people outside of threadly, and is thus why it is package protected 
- * visibility.</p>
+ * <p>Abstract implementation for implementations of {@link PrioritySchedulerService}.  In 
+ * general this wont be useful outside of Threadly developers, but must be a public interface 
+ * since it is used in sub-packages.</p>
+ * 
+ * <p>If you do find yourself using this class, please post an issue on github to tell us why.  If 
+ * there is something you want our schedulers to provide, we are happy to hear about it.</p>
  * 
  * @author jent - Mike Jensen
  * @since 4.3.0
  */
 @SuppressWarnings("deprecation")
-abstract class AbstractPriorityScheduler extends AbstractSubmitterScheduler 
-                                         implements PrioritySchedulerInterface {
+public abstract class AbstractPriorityScheduler extends AbstractSubmitterScheduler 
```

Comment:
Visibility had to increase for TestableScheduler to reuse some of the logic here.  Sadly that means this will also be in the javadocs :-(